readme.md

```diff
@@ -238,44 +238,44 @@ see: [source](https://github.com/pelias/interpolation/blob/master/cmd/server.js)
 # docker
 
 ### build docker image
-this can take some time for the first build due to installing libpostal from source
+This can take some time for the first build due to installing libpostal from source.
 ```bash
 docker build -t pelias/interpolation .
 ```
 
-you can confirm that worked with:
+You can confirm that worked with:
 ```bash
 docker images
 REPOSITORY             TAG                 IMAGE ID            CREATED             SIZE
 pelias/interpolation   latest              7ca651b86a63        16 minutes ago      3.068 GB
 ```
 
 ### run docker image
-this will run a new container based off the image created above
+This will run a new container based off the image created above.
 
-notes:
+Notes:
 - `-p` controls port mapping (port `3000` in the container maps to `5000` in the host)
-- `-v` controls volume mapping (`/data` in the container maps to `/data` in the host)
+- `-v` controls volume mapping (`/data` in the container maps to `/tmp` in the host)
```

Comment:
Could we please change the order of words in the sentence so it matches the order used by docker, so it should read:

> `/tmp` on the host maps to `/data` in the container